bundles/ranvier-combat/behaviors/item/decay.js

```diff
@@ -37,8 +37,12 @@ module.exports = srcPath => {
         }
 
         if (belongsTo) {
+          const owner = this.findOwner();
+          if (owner) {
+            Broadcast.sayAt(owner, `Your ${this.name} has rotted away!`);
+            Broadcast.prompt(owner);
```

Comment:
I generally only prompt the player if their stats have changed and need to see them or if the output actually requires feedback.